docs/mosaic.md

```diff
@@ -17,10 +23,11 @@ Because user might want to choose which pixel goes on top of the tile, this plug
 `rio_tiler.mosaic.mosaic_reader(assets, tiler, *args* pixel_selection=None, chunk_size=None, Threads=10, **kwargs)`
 
 Inputs:
+
 - assets : list, tuple of rio-tiler compatible assets (url or sceneid)
-- tiler: Rio-tiler's tiler function (e.g rio_tiler.landsat8.tile) 
+- tiler: Rio-tiler's tiler function (e.g rio_tiler.landsat8.tile)
```

Comment:
can we replace this by `tiler: Callable that returns a tuple of numpy.array (e.g tile, mask = rio_tiler.reader.tile(x, y, z, **kwargs)`